package.json

```
@@ -12,7 +12,7 @@
     "flow": "flow",
     "build": "rm -rf ./vendor && mkdir -p ./vendor/braintree-web && cp -R ~/bt/braintree.js/dist/npm/* ./vendor/braintree-web",
     "lint": "eslint src/ test/ *.js",
-    "test": "npm run lint && npm run flow-typed && npm run flow && npm run karma",
+    "test": "npm run flow-typed && npm run flow && npm run karma",
```

This makes it so the lint command does not get run at all via CI. I'd suggest. If you don't want it to run before every test, I'd suggest:

```suggestion
    "test": "npm run flow-typed && npm run flow && npm run karma",
    "postTest": "npm run lint"
```

So it runs after the tests complete as a final test.

---

sorry, it was a mistake. I put it back